src/guide/android_troubleshooting.md

```diff
@@ -12,7 +12,9 @@ Starting with this 1.8.20 release, the Kotlin K2 compiler has a preview version
 
 ## JNA library
 
-JNA relies on specific class and method names to load native libraries and access native code. If these names are obfuscated or removed, it can cause runtime errors and failures in native library loading.
+JNA relies on specific class and method names to load native libraries and access native code. If these names are obfuscated or removed, it can cause runtime errors and failures in native library loading. 
+
+The JNA library code obfuscation issue may not be apparent until compiled on "release" mode with maximum optimization and minimal footprint size that'll be used when deploying and publishing an application.
```

Comment:
What would be a more comprehensive word for `release` mode here? 
As the wording changes from platform to platform, e.g.; on React Native it's named `production` AFAIK

Reply:
Nevermind, it's OK to use `release` here as it's the wording Android documentation uses.

Reply:
I think `release` should be pretty well understood and the same release config would need to be applied to Android also